macroid-core/src/main/scala/macroid/Contexts.scala

```diff
@@ -51,7 +52,6 @@ object ContextWrapper {
 If you are inside Activity or Fragment, extend Contexts[Activity] or Contexts[Fragment],
 otherwise pass an instance of `FragmentManagerContext` from outside.
 Please note that for support fragments you need to extends Contexts[FragmentActivity]""")
-/** FragmentManager context */

```

Comment:
I thought it was unintentional, since it doesn't add any value. It's almost the same as the class name.